css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,46 @@
+button {
+  padding: 13px 28px;
+  font-size: 19px;
+}
+
+button:hover {
+  background-color: black;
+  color: white;
+  cursor: pointer;
+}
+
+thead > tr {
+  background: #000;
+  font-size: 16px;
+  color: white;
+}
+
+tbody > tr:nth-child(1) {
```

Comment:
while this works it would be better to take advantage of `nth-child`s ability to take an argument like `odd`

tbody > tr:nth-child(odd)

so keep playing with that and we will look at it in the Q and A!